client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -596,19 +612,20 @@ $.entwine('ss', function($){
     },
     showGroupsList: function (instant) {
       const holder = this.closest('.field');
-      this.showListElement(holder, holder.next().filter('.treemultiselect'), instant);
+      console.log(holder);
```

Comment:
Need to remove the console.log statement

Reply:
Oops yup. Done.